extra/modules/confiant-ad-quality/src/main/java/org/prebid/server/hooks/modules/com/confiant/adquality/v1/ConfiantAdQualityBidResponsesScanHook.java

```diff
@@ -60,7 +60,8 @@ public Future<InvocationResult<AllProcessedBidResponsesPayload>> call(
         final BidRequest bidRequest = getBidRequest(auctionInvocationContext);
         final List<BidderResponse> responses = allProcessedBidResponsesPayload.bidResponses();
         final Map<Boolean, List<BidderResponse>> needScanMap = responses.stream()
-                .collect(Collectors.groupingBy(bidderResponse -> !biddersToExcludeFromScan.contains(bidderResponse.getBidder())));
+                .collect(Collectors.groupingBy(bidderResponse -> !biddersToExcludeFromScan.contains(bidderResponse.getBidder())
+                        && !bidderResponse.getSeatBid().getBids().isEmpty()));
```

Comment:
I suggest extracting this condition to a separate method to make it clear

the name could be `isRequiredToScan(bidderResponse)` or something similar to that